doc/reference/configuration/configuration_reference.rst

```diff
@@ -221,6 +488,718 @@ This section describes options related to :ref:`storing configuration in etcd <c
+
+    See also: :ref:`database.instance_uuid <configuration_reference_database_instance_uuid>`
+
+    | Type: string
+    | Default: box.NULL
+    | Environment variable: TT_DATABASE_REPLICASET_UUID
+
+
+.. _configuration_reference_database_txn_isolation:
+
+.. confval:: database.txn_isolation
+
+    A transaction :ref:`isolation level <transaction_model_levels>`.
+
+    | Type: string
+    | Default: ``best-effort``
+    | Possible values: ``best-effort``, ``read-committed``, ``read-confirmed``
+    | Environment variable: TT_DATABASE_TXN_ISOLATION
+
+
+.. _configuration_reference_database_txn_timeout:
+
+.. confval:: database.txn_timeout
+
+    A timeout (in seconds) after which the transaction is rolled back.
+
+    See also: :ref:`box.begin() <box-begin>`
+
+    | Type: number
+    | Default: 3153600000
+    | Environment variable: TT_DATABASE_TXN_TIMEOUT
+
+
+.. _configuration_reference_database_use_mvcc_engine:
+
+.. confval:: database.use_mvcc_engine
+
+    Whether the :ref:`transactional manager <txn_mode_transaction-manager>` is enabled.
+
+    | Type: boolean
+    | Default: false
+    | Environment variable: TT_DATABASE_USE_MVCC_ENGINE
+
+
+
+
+
+..  _configuration_reference_iproto:
+
+iproto
+------
+
+The ``iproto`` section is used to configure parameters related to communicating to and between cluster instances.
+
+.. NOTE::
+
+    ``iproto`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+-   :ref:`iproto.advertise.* <configuration_reference_iproto_advertise>`
+
+    -   :ref:`iproto.advertise.client <configuration_reference_iproto_advertise_client>`
+    -   :ref:`iproto.advertise.peer <configuration_reference_iproto_advertise_peer>`
+    -   :ref:`iproto.advertise.sharding <configuration_reference_iproto_advertise_sharding>`
+
+-   :ref:`iproto.listen <configuration_reference_iproto_listen>`
+-   :ref:`iproto.net_msg_max <configuration_reference_iproto_net_msg_max>`
+-   :ref:`iproto.readahead <configuration_reference_iproto_readahead>`
+-   :ref:`iproto.threads <configuration_reference_iproto_threads>`
+
+
+..  _configuration_reference_iproto_advertise:
+
+iproto.advertise.*
+~~~~~~~~~~~~~~~~~~
+
+The ``iproto.advertise.*`` configuration options specify URIs used to advertise the current instance to other cluster members or clients.
+These options accept an URI in the following formats:
+
+-   User :ref:`credentials <configuration_reference_credentials>` and an address: ``username@host:port`` or ``username:password@host:port``.
+
+-   User credentials: ``username@`` or ``username:password@``.
+    In this case, an advertise address is taken from :ref:`iproto.listen <configuration_reference_iproto_listen>`.
+
+-   An address: ``host:port``.
+
+If ``password`` is missing, it is taken from :ref:`credentials <configuration_reference_credentials>` for the specified ``username``.
+
+You can also use a Unix domain socket (``unix/:``) instead of ``host:port``.
+
+.. NOTE::
+
+    The ``host`` value cannot be ``0.0.0.0``.
```

Comment:
It also can't be `[::]`. And the port can't be zero. See [here](https://github.com/tarantool/tarantool/blob/2c422a0b98675cab480ebb44256dfeb72bd2f820/src/box/lua/config/instance_config.lua#L686-L687).